flake.nix

```diff
@@ -61,6 +61,14 @@
                 project.hsPkgs.hspec-discover.components.exes."hspec-discover"
                 project.hsPkgs.markdown-unlit.components.exes."markdown-unlit"
               ];
+              #   FIXME (peter-mlabs): This is currently broken.
```

Comment:
nit; Syntax is `FIXME(peter-mlabs): This is currently broken.` (without the space)

Also FIXME would indicate there's a behavior in need of fixing. Perhaps TODO is more appropriate? As we want to add a new behavior.

Reply:
Wait, there's standard syntax around this?? I've always done this informally! 

What is the spec for this?

Reply:
Lol spec! The only thing close to that I've seen was https://peps.python.org/pep-0350/.

I'm hoping to adopt things like https://github.com/pgilad/leasot and have up to date reports with codetags.

`<TODO|FIX|WARN|HACK|WHATEVER>(<username|bug link>): <Description>`

That's what I've seen used in bigger codebases. Emacs magit-todos for example is excellent in representing this.